bin/authority-service/README.md

```diff
@@ -0,0 +1,49 @@
+# Authority Service
+This is a microservice for substrate nodes, which handle authorization process. Nodes using the API can request service to get information if they are authorized or not. API provides two simple requests:
+* slot number authorization:
+* node name authorization:
+
+## Build
+```
+cargo build -p authority-service
+```
+
+## Run
+```
+./target/debug/authority-service <path-to-config-file.json>
```

Comment:
Seem's like this command doesn't work

```
% ./target/debug/authority-service config/config.json
zsh: no such file or directory: ./target/debug/authority-service
```

Maybe just  `cargo run -- config/config.json` ?

Reply:
I've changed this to run optional without configuration file.

Reply:
Error says there is no binary, not configuration file 🙃

Reply:
`cargo +nightly run` works for me